DigitalLearningSolutions.Web/Views/LearningPortal/SelfAssessments/SelfAssessmentOverview.cshtml

```diff
@@ -1,103 +1,104 @@
-    }
-
-  </div>
-}
-
+﻿@inject IConfiguration Configuration
+@using DigitalLearningSolutions.Data.Helpers
+@using DigitalLearningSolutions.Web.ViewModels.LearningPortal.SelfAssessments
+@using Microsoft.Extensions.Configuration
+@model SelfAssessmentOverviewViewModel
+
+@{
+  Layout = "SelfAssessments/_Layout";
+  ViewData["Title"] = "Self Assessment";
+  ViewData["SelfAssessmentTitle"] = Model.SelfAssessment.Name;
+}
+
+@section breadcrumbs {
+  <li class="nhsuk-breadcrumb__item">
+    <a class="nhsuk-breadcrumb__link" asp-action="SelfAssessment" asp-route-selfAssessmentId="@Model.SelfAssessment.Id">@Model.SelfAssessment.Name</a>
+  </li>
+  <li class="nhsuk-breadcrumb__item">@Model.VocabPlural()</li>
+}
+
+@section mobilebacklink
+{
+  <p class="nhsuk-breadcrumb__back">
+    <a class="nhsuk-breadcrumb__backlink" asp-action="SelfAssessment"
+       asp-route-selfAssessmentId="@Model.SelfAssessment.Id">
+      Back to @Model.SelfAssessment.Name
+    </a>
+  </p>
+}
+
+<link rel="stylesheet" href="@Url.Content("~/css/learningPortal/selfAssessment.css")" asp-append-version="true">
+<h1>@Model.SelfAssessment.Name - @Model.VocabPlural()</h1>
+@if (Model.NumberOfOptionalCompetencies > 0) {
+  <a class="nhsuk-button nhsuk-button--secondary trigger-loader" asp-route-selfAssessmentId="@Model.SelfAssessment.Id" asp-route-vocabulary="@Model.VocabPlural()" asp-action="ManageOptionalCompetencies">
+    Manage optional @Model.VocabPlural().ToLower()
+  </a>
+}
+@if (Model.SelfAssessment.IsSupervised) {
+  <a class="nhsuk-button nhsuk-button--secondary trigger-loader" asp-route-selfAssessmentId="@Model.SelfAssessment.Id" asp-action="StartRequestVerification">
+    Request @Model.SelfAssessment.Vocabulary.ToLower() verification
+  </a>
+}
+@if (Model.CompetencyGroups.Any()) {
+  foreach (var competencyGroup in Model.CompetencyGroups) {
+    <div class="nhsuk-panel competency-group-panel nhsuk-u-padding-0">
+      <details class="nhsuk-details nhsuk-expander" @(Model.CompetencyGroups.Count() == 1 ? "open" : ViewContext.RouteData.Values.ContainsKey("competencyGroupId") ? ViewContext.RouteData.Values["competencyGroupId"].Equals(competencyGroup.First().CompetencyGroupID.ToString()) ? "open" : "" : "")>
+        <summary class="nhsuk-details__summary">
+          <span class="competency-group-title nhsuk-details__summary-text">
+            @competencyGroup.Key
+          </span>
+        </summary>
+        <div class="nhsuk-details__text">
+          @if (Model.SelfAssessment.LinearNavigation) {
+            <partial name="SelfAssessments/_MeanScores"
+                     view-data="@(new ViewDataDictionary(ViewData) { { "competencyGroup", competencyGroup } })" />
+          }
+
+          <partial name="SelfAssessments/_OverviewTable" view-data="@( new ViewDataDictionary(ViewData) { { "linearNavigation", Model.SelfAssessment.LinearNavigation }, { "selfAssessment", Model.SelfAssessment } })" model="competencyGroup" />

```

Comment:
Some more long lines to break if possible

Reply:
Broken these down within reason, there are still some pretty long lines.